app/components/publications/publication/publication-navigation.hbs

```diff
@@ -0,0 +1,25 @@
+<WebComponents::AuTabs @reversed="true">
+  <WebComponents::AuTab @isHierarchicalBack="true" @route="publications"
+                        data-test-publication-case-nav-go-back></WebComponents::AuTab>
+  <WebComponents::AuTab @route="publications.publication.case" data-test-publication-case-nav-case>
+    {{t "agendaitem-case"}}
+  </WebComponents::AuTab>
+  {{#unless this.loadData.isRunning}}
```

Comment:
Gezien enkel de `documentsCount` afhankelijk is van `loadData`, is deze hele block conditioneel renderen wat overkill. Als je deze conditie puur rond `<p>{{this.documentsCount}}</p>` zet, wordt de tab al van in den beginne gerendered, en komt het getal er gewoon bij. Dat is voor het oog ook rustiger dan een hele tab die er na een halve seconde tussenflitst.